docs/utilities/batch.md

```diff
@@ -1,460 +1,322 @@
-        plugins {
-            id 'java'
-            id 'io.freefair.aspectj.post-compile-weaving' version '8.1.0'
-        }
-        
-        repositories {
-            mavenCentral()
-        }
-        
-        dependencies {
-            aspect 'software.amazon.lambda:powertools-sqs:{{ powertools.version }}'
-        }
-        
-        sourceCompatibility = 11 // or higher
-        targetCompatibility = 11 // or higher
     ```
 
-=== "Gradle Java 1.8"
+=== "Gradle"
 
-    ```groovy hl_lines="3 11"
-        plugins {
-            id 'java'
-            id 'io.freefair.aspectj.post-compile-weaving' version '6.6.3'
-        }
+    ```groovy
         
         repositories {
             mavenCentral()
         }
         
         dependencies {
-            aspect 'software.amazon.lambda:powertools-sqs:{{ powertools.version }}'
+            aspect 'software.amazon.lambda:powertools-batch:{{ powertools.version }}'
         }
-        
-        sourceCompatibility = 1.8
-        targetCompatibility = 1.8
     ```
+## Getting Started
 
-## IAM Permissions
+For this feature to work, you need to **(1)** configure your Lambda function event source to use `ReportBatchItemFailures`,
+and **(2)** return [a specific response](https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html#services-sqs-batchfailurereporting){target="_blank" rel="nofollow"}
+to report which records failed to be processed.
 
-This utility requires additional permissions to work as expected. Lambda functions using this utility require the `sqs:DeleteMessageBatch` permission.
+You use your preferred deployment framework to set the correct configuration while this utility handles the correct response to be returned.
 
-If you are also using [nonRetryableExceptions](#move-non-retryable-messages-to-a-dead-letter-queue) attribute, utility will need additional permission of `sqs:GetQueueAttributes` on source SQS. 
-It also needs `sqs:SendMessage` and `sqs:SendMessageBatch` on configured dead letter queue.  
+A complete [Serverless Application Model](https://aws.amazon.com/serverless/sam/) example can be found
+[here](https://github.com/aws-powertools/powertools-lambda-java/tree/main/examples/powertools-examples-batch) covering
+all of the batch sources.
 
-If source or dead letter queue is configured to use encryption at rest using [AWS Key Management Service (KMS)](https://aws.amazon.com/kms/), function will need additional permissions of 
-`kms:GenerateDataKey` and `kms:Decrypt` on the KMS key being used for encryption. Refer [docs](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-key-management.html#compatibility-with-aws-services) for more details.
 
-Refer [example project](https://github.com/aws-samples/aws-lambda-powertools-examples/blob/main/java/SqsBatchProcessing/template.yaml#L105) for policy details example.
 
+!!! note "You do not need any additional IAM permissions to use this utility, except for what each event source requires."
```

Comment:
you tried and everything is ok without SQS permissions ?

what about KMS if the queue is encrypted withit ?

Reply:
I stole this from [Heitor's new changes for python](https://github.com/aws-powertools/powertools-lambda-python/pull/2857) . I think deferring to the regular integration requirements here keeps us DRY - PT doesn't add any extra demands on what you need anyway to use e.g. SQS+KMS with Lambda batch